azure-pipelines.yml

```diff
@@ -150,7 +150,7 @@ stages:
               # We repull the develop branch. We could build the local branch that is already present but we should
               # prune it's depth since we don't need the git history.
               CMD="docker buildx build -f docker/Dockerfile.tests --tag greatexpectations/test:${DOCKER_TAG} "
-              CMD+="--target azure --build-arg PYTHON_VERSION=$(python.version) "
```

Comment:
I remove the `azure` target since we don't want this environment to be special in any way so we change this to the `test` target.